routes/users.js

```diff
@@ -10,5 +10,15 @@ router.get("/users", async (req, res) => {
   res.send(users);
 });
 
+router.get("/users/:id", async (req, res) => 
+  {
+    const userid = parseInt(req.params.id);
+    if(isNaN(userid)) {
+      res.send({"error": "Specified User not found"});
+    }
+    var user = users.find(item => item.id === userid);
```

Comment:
Can we change the `var` to a `const` since the value for this variable won't change?